course_catalog/etl/ocw_next.py

```diff
@@ -111,6 +119,11 @@ def transform_resource(s3_key, resource_data, s3_resource, force_overwrite):
     if not file_s3_path:
         return
 
+    title = resource_data.get("title")
+
+    if title == "3play caption file" or title == "3play pdf file":
+        return
```

Comment:
Skip 3play files?

Reply:
Yes - I don't think we want the search to return 3play caption and transcript files separately from videos.